python/Ganga/Core/GangaRepository/VStreamer.py

```diff
@@ -361,7 +364,9 @@ def end_element(name):
                 # unescape the special characters
                 s = unescape(self.value_construct)
                 #logger.debug('string value: %s',s)
-                val = eval(s, config_scope)
+                if s not in _cached_eval_strings:
+                    _cached_eval_strings[s] = eval(s, getProxyInterface().__dict__)
+                val = copy.deepcopy(_cached_eval_strings[s])
```

Comment:
You need access to things like `datetime` to evaluate JobTime.
Which reminds me I planned to add a comment to this effect here.